x/cork/keeper/test_common.go

```diff
@@ -191,6 +191,7 @@ var (
 )
 
 // TestInput stores the various keepers required to test gravity
+// TODO This file is mostly unused. Ask Eric/Collin about whether it's needed.
```

Comment:
yeah I think we should be able to remove all or most of this with the introduction of the generated mocks